lib/blockscore/base.rb

```diff
@@ -33,12 +33,12 @@ def inspect
 
     def save
       save!
-    rescue BlockScore::Error
+    rescue Error
       false
     end
 
     def save!
-      fail BlockScore::Error, "#{resource} is already deleted" if deleted?
+      assert_not_deleted
       capture_attributes(post(endpoint, attributes))
```

Comment:
Thanks for that clarification. I didn't see it as a flaw in mutant. Perhaps the best approach is to move ALL the unit tests to specs/integration and then selectively move/write unit tests for these without VCR. My struggle here is there is a limited contract between the external service and the API and the API and the client software. So for instance a post without content is legitimate - not saying nil is - but certainly posting empty attributes are. 

I have not looked at this mutation coverage as of yet. I did not expect it to be without another pass.